associacao-brasileira-de-normas-tecnicas-ufrgs-note-initials-with-ibid.csl

```diff
@@ -1,275 +1,485 @@
-        <text macro="title"/>
-      </substitute>
+  <macro name="interviewer">
+    <!-- Macro responsavel por mostrar o nome do entrevistador -->
+    <text term="interviewer" suffix=": " text-case="capitalize-first"/>
+    <names variable="interviewer" delimiter="; ">
+      <name delimiter=" e " sort-separator=" " delimiter-precedes-last="always"/>
+      <et-al font-style="italic"/>
     </names>
   </macro>
-  <!-- :: Endereço html :: -->
-  <macro name="access">
-    <text variable="URL" prefix=" Disponível em: &lt;" suffix="&gt;"/>
-    <date variable="accessed" prefix=". Acesso em: " suffix=".">
-      <date-part name="day" suffix=" "/>
-      <date-part name="month" form="short" suffix=". " text-case="lowercase"/>
-      <date-part name="year"/>
-    </date>
+  <macro name="recipient">
+    <!-- Macro responsavel por mostrar o nome do destinatário -->
+    <text term="recipient" suffix=": " text-case="capitalize-first"/>
+    <names variable="recipient" delimiter="; ">
+      <name delimiter=" e " sort-separator=" " delimiter-precedes-last="always"/>
+      <et-al font-style="italic"/>
+    </names>
   </macro>
-  <!-- :: Área do título :: -->
-  <macro name="title">
+  <macro name="translator">
+    <!-- Macro responsavel por mostrar o nome do tradutor -->
+    <text term="translator" suffix=": " text-case="capitalize-first"/>
+    <names variable="translator" delimiter=", ">
+      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"/>
+      <et-al font-style="italic"/>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <!-- Macro responsável por mostrar o autor na citação -->
     <choose>
-      <if type="chapter article-newspaper legislation article-magazine article-journal bill entry-encyclopedia paper-conference" match="any">
-        <text variable="title"/>
+      <if type="broadcast motion_picture song" match="any">
+        <text variable="title" form="short" text-case="uppercase"/>
       </if>
-      <else-if type="book thesis patent report" match="any">
-        <text variable="title" font-weight="bold"/>
-      </else-if>
-      <else-if type="dataset" match="any">
-        <text variable="title" suffix=". "/>
-      </else-if>
       <else>
-        <text variable="title" font-weight="bold"/>
+        <names variable="author">
+          <name form="short" delimiter="; " delimiter-precedes-last="never" initialize-with=". " name-as-sort-order="all">
+            <name-part name="family" text-case="uppercase"/>
+            <name-part name="given" text-case="capitalize-first"/>
+          </name>
+          <et-al font-style="italic"/>
+          <substitute>
+            <names variable="editor"/>
+            <names variable="translator"/>
+            <choose>
+              <if type="book">
+                <!-- uso do título quando não houver autor -->
+                <text variable="title" form="short" text-case="uppercase"/>
+              </if>
+            </choose>
```

Comment:
```suggestion
            <!-- uso do título quando não houver autor -->
            <text variable="title" form="short" text-case="uppercase"/>
```

I meant this: you don't need a conditional at all: you always wont the shor title in uppercase, regardless of item type (that's at least what the previous version did; this version only uses it for books, which I'm guessing isn't what you want)